packages/core/src/Camera.ts

```diff
@@ -692,6 +724,13 @@ export class Camera extends Component {
     this._invViewProjMat = null;
   }
 
+  /**
+   * @internal
+   */
+  _getInternalColorTextureFormat(): TextureFormat {
+    return this._enableHDR ? TextureFormat.R16G16B16A16 : TextureFormat.R8G8B8A8;
+  }
```

Comment:
Are you sure this is the best format choice, for example `R11F_G11F_B10F` you have tryed?

Reply:
done